README.md

```diff
@@ -74,39 +44,167 @@ This program was created with the following tech stack and packages:
 
 Tested remotely via an AWS ec2 instance.
 
-
 ## Setup
 
+If the API is not yet integrated, you can still use the API locally on your machine using the following steps:
+
 1. Install [Node.Js](https://nodejs.org/en/)
-2. Download the repository
-3. Receive Kukarella files
+2. Install [ffmpeg](https://ffmpeg.org/)
+   - We recommend following the steps on [adamtheautomator.com](https://adamtheautomator.com/install-ffmpeg/) for your
+   corresponding operating system. We also recommend the Powershell approach for Windows users.
```

Comment:
I wonder if eventually we should just write out the steps from this tutorial?

Reply:
I vote that we don't because installing ffmpeg is just a pre-requisite for our program and I feel like writing down all these instructions for the different OS's will take a lot more space than it should

Reply:
You're also very right on that